r2dbc/mysql/src/test/java/com/google/cloud/sql/core/R2dbcMysqlIamAuthIntegrationTests.java

```diff
@@ -89,53 +86,17 @@ public void setUpPool() {
     this.connectionPool = new ConnectionPool(configuration);
     // [END cloud_sql_connector_mysql_r2dbc_iam_auth]
 
-    this.tableName = String.format("books_%s", UUID.randomUUID().toString().replace("-", ""));
-
-    // Create table
-    Mono.from(this.connectionPool.create())
-        .flatMapMany(
-            c ->
-                c.createStatement(
-                        String.format("CREATE TABLE %s (", this.tableName)
-                            + "  ID CHAR(20) NOT NULL,"
-                            + "  TITLE TEXT NOT NULL"
-                            + ")")
-                    .execute())
-        .blockLast();
-  }
-
-  @After
-  public void dropTableIfPresent() {
-    String dropStmt = String.format("DROP TABLE %s", this.tableName);
-    Mono.from(this.connectionPool.create())
-        .delayUntil(c -> c.createStatement(dropStmt).execute())
-        .block();
   }
 
   @Test
   public void pooledConnectionTest() {
-    String insertStmt = String.format("INSERT INTO %s (ID, TITLE) VALUES (?, ?)", this.tableName);
-    Mono.from(this.connectionPool.create())
-        .flatMapMany(
-            c ->
-                c.createStatement(insertStmt)
-                    .bind(0, "book1")
-                    .bind(1, "Book One")
-                    .add()
-                    .bind(0, "book2")
-                    .bind(1, "Book Two")
-                    .execute())
-        .flatMap(result -> result.map((row, rowMetadata) -> row.get(0)))
-        .blockLast();
-
-    String selectStmt = String.format("SELECT TITLE FROM %s ORDER BY ID", this.tableName);
-    List<String> books =
+    List<Object> rows =
         Mono.from(this.connectionPool.create())
-            .flatMapMany(connection -> connection.createStatement(selectStmt).execute())
-            .flatMap(result -> result.map((r, meta) -> r.get("TITLE", String.class)))
+            .flatMapMany(connection -> connection.createStatement("SELECT NOW() as TS").execute())
+            .flatMap(result -> result.map((r, meta) -> r.get("TS", Object.class)))
```

Comment:
`r.get("TS", Timestamp.class)`?

Reply:
It's not a Timestamp class actually. If this were a JDBC driver, it would be. However in R2DBC who knows what it is.

Reply:
Did you try `Instant.class`? Googling and that seems to work.

Reply:
I have now updated it to use Instant.